docker-compose.yml

```diff
@@ -84,11 +97,13 @@ services:
       - MYSQL_PASSWORD=mysqlpw
 
   redis:
+    logging: *loki-logging
```

Comment:
See other comments - the use of loki-logging needs to be optional.
is there an alternative to hardwiring `logging: *loki-logging`, e.g. 

* Configure using environment variable
* Use promtail?

Reply:
Promtail uses Kubernetes service discovery to associate the log files with the services. In the docker/docker-compose world this is done by the loki logging driver plugin (https://grafana.com/docs/loki/latest/clients/docker-driver/).
Moved the logging driver configuration from the docker-compose file to the docker daemon (/etc/docker/daemon.json) as explained in https://grafana.com/docs/loki/latest/clients/docker-driver/configuration/. This will be a manual step in addition to the installation of the grafana/loki-docker-driver plugin.

Reply:
Having to configure the Docker daemon sounds problematic!

Why not just use `logging: ${EVENTUATE_EXAMPLE_LOGGING_DRIVER:-json-file}`

Reply:
@senthilcool What about this?

Reply:
Will try this out.

Reply:
@cer The loki logging driver configuration includes the loki-url as part of the options. As this option is not applicable for json-file, it needs to be included conditionally. 
```
    driver: loki
    options:
      loki-url: "http://localhost:3100/loki/api/v1/push"
```
Haven't figured out a way to do this. Let me know if you have any suggestions.

Reply:
two options come to mind:

1. What happens if you configure json-file with loki-url - does it fail?
2. copy/paste the service definitions into docker-compose-observability.yml

Reply:
@cer 
1. Yes, json-file with loki-url failed and so was not an option.
2. Copy/pasted the service definitions into docker-compose-observable.yml.